codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieEncoder.java

```diff
@@ -68,6 +68,7 @@ public static String encode(Cookie cookie) {
 
         if (cookie.maxAge() != Long.MIN_VALUE) {
             add(buf, CookieHeaderNames.MAX_AGE, cookie.maxAge());
+            add(buf, CookieHeaderNames.EXPIRES, HttpHeaderDateFormat.get().format(cookie.maxAge() + System.currentTimeMillis()));
```

Comment:
Comments say units on `cookie.maxAge()` is [seconds](https://github.com/netty/netty/blob/master/codec-http/src/main/java/io/netty/handler/codec/http/Cookie.java#L105).  Forgive me if I don't have enough context but is the intention to add `seconds + milli seconds`?